src/poly/commitment/verifier.rs

```diff
@@ -162,9 +162,9 @@ impl<C: CurveAffine> Proof<C> {
         let c_packed = transcript.squeeze().get_lower_128();
         let c: C::Scalar = get_challenge_scalar(Challenge(c_packed));
 
-        // Check
-        // [c] P + [c * v] U + [c] sum(L_i * u_i^2) + [c] sum(R_i * u_i^-2) + delta - [z1] G - [z1 * b] U - [z1 - z2] H
-        // = 0
+        // Construct
+        // [c] P + [c * v] U + [c] sum(L_i * u_i^2) + [c] sum(R_i * u_i^-2) + delta - [z1 * b] U + [z1 - z2] H
+        // = [z1] (G + H)
```

Comment:
Say where the equality is checked.

Reply:
The comment is correct to note that we've modified where the actual computation of `[z1] (G + H)` happens. If we just add a detail here that "the computation of `[z1] (G + H)` happens in either `Guard::use_challenges` or `Guard::use_g`" that would be cool.

Reply:
I double-checked and yeah, good catch on the `+ [z1 - z2] H` thing.